### PR TITLE
Instruments and director fix

### PR DESCRIPTION
**Pull request by Markvanderuit**
- Fixes issue with director timelines loading incorrectly.
- Fixes issue with instruments loading incorrectly.
- Instruments now displayed in timeline blocks.
- Instruments now displayed in timeline toast.

### CODE REVIEW

File: objects/ProjectManager.js [hunk 1]
```diff
@@ -81,6 +81,7 @@ class ProjectManager {
                     const directorTimelineXML = result.scriptingProject.directorTimeline[0];
                     this.data.scriptingProject.directorTimeline =
                         DirectorTimeline.fromXML(directorTimelineXML);
+                    console.log(this.data.scriptingProject.directorTimeline);
```
Comment (Markvanderuit) [inline objects/ProjectManager.js]: Removing...

Comment (martijnjanssen) [inline objects/ProjectManager.js]: :+1:

File: elements/timeline-view/timeline-block.html [hunk 2]
```diff
@@ -71,12 +71,21 @@
                 <span>{{block_data.beginCount}} - {{block_data.endCount}}</span>
             </div>
             <div class="horizontal-separator"></div>
-            <div class="card-content" style="background-color: transparent; height: 100%">
+            <div class="card-content" style="background-color: transparent;">
```
Comment (martijnjanssen) [inline elements/timeline-view/timeline-block.html]: Could this be moved to the styling section.

File: elements/timeline-view/timeline-toast.html [hunk 3]
```diff
@@ -75,6 +75,13 @@
                     <span class="content-text">{{block_data.description}}</span>
                 </div>
                 <div></div>
+                <div>
+                    <span class="content-title">Instruments</span>
+                    <template is="dom-repeat" items="{{block_data.instruments}}">
+                        <span class="content-text" style="padding-right: 20px;">{{item.name}}</span>
```
Comment (martijnjanssen) [inline elements/timeline-view/timeline-toast.html]: Could this also go in the style section?

File: elements/timeline-view/timeline-block.html (hunk 2 above)
Comment (Markvanderuit) [inline elements/timeline-view/timeline-block.html]: Moving...

File: elements/timeline-view/timeline-toast.html (hunk 3 above)
Comment (Markvanderuit) [inline elements/timeline-view/timeline-toast.html]: Ditto...

**Comment by martijnjanssen** [comment]
Looks good!

**Comment by mennoo1996** [comment]
Nice!

**Pull request merged**